.gitignore

```diff
@@ -7,7 +7,7 @@ build
 node_modules
 tmp
 .vscode/*
-.env
+.env.local
```

Comment:
nit: generally `.env` is explicitly ignored with instructions on `.env.local` or any `.env.*` to be copied to `.env`

Reply:
See https://github.com/FuelLabs/forc.pub/pull/22#discussion_r1912559437